devices.js

```diff
@@ -5322,6 +5322,20 @@ const devices = [
         description: 'Plug Shine Zigbee controller',
         extend: generic.light_onoff_brightness,
     },
+    {
+        zigbeeModel: ['RemoteControl '],
+        model: '500.67',
+        vendor: 'Paulmann',
+        description: 'RGB remote control',
+        supports: 'action',
+        fromZigbee: [
+            fz.command_on, fz.command_off, fz.command_toggle, fz.command_step,
+            fz.command_move_to_color_temp, fz.command_move_to_color, fz.command_stop,
+            fz.command_move, fz.command_color_loop_set, fz.command_ehanced_move_to_hue_and_saturation,
```

Comment:
is there any way to deal with the genOta messages?

Reply:
We don't have to deal with them as we don't have an image.